pages/home.html

```diff
@@ -2,9 +2,47 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="../css/bootstrap.css">
+    <link rel="stylesheet" href="../css/master.css">
+    <script src="https://kit.fontawesome.com/6854d7f83d.js" crossorigin="anonymous"></script>
     <title>Home Page</title>
   </head>
-  <body>
+
+  <main id=homeBackground>
+    <body >
```

Comment:
Swap these tags around, please.

Reply:
main inside body?

Reply:
Yeah, like this:
`<body>
  <main>
    <p>hello world!</p>
  </main>
</body>`

Reply:
are the cards ok inside of <main> as well?

Reply:
main